ConsolidatedCode/Functions_Gillespie.py

```diff
@@ -11,7 +11,7 @@
  http://localhost:8888/edit/BioResearch/python/Functions.py#   NOTE: There is no checking for negative values in this version.'''
 
 
-def gillespie(reactions_list, stop_time, initial_state_vector):
+def gillespie(reactions_list: object, stop_time: object, initial_state_vector: object) -> object:
```

Comment:
The addition of ": object" was suggested by pycharm shall we keep for clarity or remove to maintain integrity of initial code.